utilities/sample-dashboard.json

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
I'm attempting to figure out the purpose of this dashboard.  It only has the download panel (which doesn't use any variables), but the dashboard defines lots of different variables.

We could create a dashboard with different types of variables defined to demonstrate how to define variables that depend upon other variables.  If we do this, then we should probably include a text panel describing the purpose of the dashboard.  (It probably won't be obvious just by looking at the variables across the top of the dashboard.)